src/invert/laplace/impls/multigrid/multigrid_alg.cxx

```diff
@@ -429,15 +429,15 @@ void MultigridAlg::pGMRES(BoutReal *sol,BoutReal *rhs,int level,int iplag) {
 
   if(level == 0) {
     if((rProcI == 0) && (pcheck == 1)) {
-      rederr = log(error/ini_e)/((double)num);
+      rederr = log(error / ini_e) / static_cast<double>(num);
```

Comment:
Whilst perhaps not relevant for this PR, should this really be `BoutReal` for consistency instead of `double`?